PhotometryScript.py

```diff
@@ -0,0 +1,164 @@
+# sum all the values of the square area selected
+####################################
+
+#print(ReferenceStarLoc, ReferenceBackgroundLoc)
+ReferenceDistance = (np.absolute(np.subtract(ReferenceStarLoc, ReferenceBackgroundLoc)))
+ReferenceRadius = np.amax(ReferenceDistance)
+print('The radius of the reference star is', ReferenceRadius)
+
+ReferenceRange = np.arange(-ReferenceRadius,ReferenceRadius+1,1)
+XReferenceRange = ReferenceStarLoc[0] + ReferenceRange
+YReferenceRange = ReferenceStarLoc[1] + ReferenceRange
+
+FullReferenceRange=[]
+for i in range(0,2*ReferenceRadius+1):
+    ColumnShape = []
+    ColumnValues = ((np.full((2*ReferenceRadius+1), YReferenceRange[i])))
+    ColumnShape  = np.column_stack((XReferenceRange,ColumnValues))
+    FullReferenceRange.append(ColumnShape)
+
+FullReferenceRange = np.array(np.concatenate((np.array(FullReferenceRange))))
+#FullestRange looks gross, but this was the only way to get it in one long 121 by 2 matrix
+#But yes, it is gross
+
+ReferenceMagValues = []
+for i in range(0,ReferenceRadius**2):
+    xVal = FullReferenceRange[i,0]
+    yVal = FullReferenceRange[i,1]
+    ReferencePixelValue=img[xVal,yVal]
+    ReferenceMagValues.append(ReferencePixelValue)
+
+ReferenceMagValues = (np.array(ReferenceMagValues)).astype(np.int16)
+ReferenceMagValues = (ReferenceMagValues - ReferenceBackgroundMag)
+ReferenceMagValues = ReferenceMagValues.clip(0)
+ReferenceMagValue = np.sum(ReferenceMagValues)
+print('The magnitude value of the reference star is',ReferenceMagValue)
+
+####################################
+#Same thing but for Object
+####################################
+
+ObjectDistance = (np.absolute(np.subtract(ObjectLoc, ObjectBackgroundLoc)))
+ObjectRadius = np.amax(ObjectDistance)
+print('The radius of the object is',ObjectRadius)
+
+ObjectRange = np.arange(-ObjectRadius,ObjectRadius+1,1)
+XObjectRange = ObjectLoc[0] + ObjectRange
+YObjectRange = ObjectLoc[1] + ObjectRange
+FullObjectRange=[]
+for i in range(0,2*ObjectRadius+1):
+    ColumnShape = []
+    ColumnValues = ((np.full((2*ObjectRadius+1), YObjectRange[i])))
+    ColumnShape  = np.column_stack((XObjectRange,ColumnValues))
+    FullObjectRange.append(ColumnShape)
+
+FullObjectRange = np.array(np.concatenate((np.array(FullObjectRange))))
+
+ObjectMagValues = []
+for i in range(0,ObjectRadius**2):
+    xVal = FullObjectRange[i,0]
+    yVal = FullObjectRange[i,1]
+    ObjectPixelValue=img[xVal,yVal]
+    ObjectMagValues.append(ObjectPixelValue)
+ObjectMagValues = (np.array(ObjectMagValues)).astype(np.int16)
+ObjectMagValues = (ObjectMagValues - ObjectBackgroundMag)
+ObjectMagValues = ObjectMagValues.clip(0)
+ObjectMagValue = np.sum(ObjectMagValues)
+print('The Magnitude Value of the object is',ObjectMagValue)
+
+####################################
+#Photometry for finding the catalog value
+####################################
+Exposure = 1
+
+InstrumentalMagnitude = -2.5*sp.log(ReferenceMagValue/Exposure)
```

Comment:
I don't think you need the exposure here as all references and objects will be taken with the same exposure. So it will be baked into the offset.